include/aspect/material_model/drucker_prager.h

```diff
@@ -76,43 +76,17 @@ namespace aspect
      * @ingroup MaterialModels
      */
     template <int dim>
-    class DruckerPrager : public MaterialModel::InterfaceCompatibility<dim>, public ::aspect::SimulatorAccess<dim>
+    class DruckerPrager : public MaterialModel::Interface<dim>, public ::aspect::SimulatorAccess<dim>
```

Comment:
Please read through the class documentation above and make sure it is still accurate after this change. If not please update, and do not forget to update the string at the bottom of the .cc file.